src/cvode/cvode_ls.c

```diff
@@ -364,20 +364,24 @@ int CVodeSetLSNormFactor(void *cvode_mem, realtype nrmfac)
   int      retval;
 
   /* access CVLsMem structure */
-  retval = cvLs_AccessLMem(cvode_mem, "CVodeSetLSNormFactor",
+  retval = cvLs_AccessLMem(cvode_mem, __LINE__, __func__,
                            &cv_mem, &cvls_mem);
   if (retval != CVLS_SUCCESS) return(retval);
 
+  SUNAssignSUNCTX(CV_SUNCTX);
+
   if (nrmfac > ZERO) {
     /* user-provided factor */
     cvls_mem->nrmfac = nrmfac;
   } else if (nrmfac < ZERO) {
     /* compute factor for WRMS norm with dot product */
-    N_VConst(ONE, cvls_mem->ytemp);
-    cvls_mem->nrmfac = SUNRsqrt(N_VDotProd(cvls_mem->ytemp, cvls_mem->ytemp));
+    SUNCheckCallLastErrNoRet(N_VConst(ONE, cvls_mem->ytemp));
+    cvls_mem->nrmfac = SUNCheckCallLastErrNoRet(N_VDotProd(cvls_mem->ytemp, cvls_mem->ytemp));
```

Comment:
We should be certain to add a comment to the developer guide that `SUNCheck` calls should not be nested inside other functions, e.g.
```
SUNRsqrt(SUNCheckCallLastErrNoRet(N_VGetLength(cvls_mem->ytemp)));
```
due to their multi-line-macro nature.